### ISSUE TITLE
1.0046: fcgi_cleanup.t may fail

### ISSUE THREAD

**Issue by eserte** [plack/Plack#611]
On one of my smokers I observe the following failure:
```
# Testing with lighttpd 1.4.35
SIG is not a valid POSIX macro at /opt/perl-5.24.1/lib/site_perl/5.24.1/FCGI/ProcManager.pm line 166

#   Failed test 'Got successful response'
#   at t/Plack-Handler/fcgi_cleanup.t line 50.

#   Failed test 'With response indicating not yet cleaned up'
#   at t/Plack-Handler/fcgi_cleanup.t line 52.
#          got: '<?xml version="1.0" encoding="iso-8859-1"?>
# <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
#          "http'
#     expected: '0'
Argument "//www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">\n<h..." isn't numeric in subtraction (-) at t/Plack-Handler/fcgi_cleanup.t line 53.

#   Failed test 'Response received within a second of being sent'
#   at t/Plack-Handler/fcgi_cleanup.t line 53.
# Looks like you failed 3 tests of 3.
t/Plack-Handler/fcgi_cleanup.t ....................... 
Dubious, test returned 3 (wstat 768, 0x300)
Failed 3/3 subtests 
```
This is a debian/jessie system. However, on another debian/jessie system with the prereqs installed (lighttpd, FCGI, FCGI::ProcManager) I cannot reproduce the failure.

**Comment by miyagawa** [comment]
Fixed in 1.0047

**Comment by skaji** [comment]
FYI: It seems that
```
SIG is not a valid POSIX macro at /opt/perl-5.24.1/lib/site_perl/5.24.1/FCGI/ProcManager.pm line 166
```
was fixed in FCGI-ProcManager-0.28.
https://metacpan.org/release/ARODLAND/FCGI-ProcManager-0.28
https://rt.cpan.org/Public/Bug/Display.html?id=120448
